constellations/gemini-user/src/afh/afh.js

```diff
@@ -0,0 +1,119 @@
+"use strict";
+require("./afh.sass");
+import React from "react";
+import API from "../utils/api.js";
+
+export class AFHPage extends React.Component {
+    state = {
+        currentTab: "Math",
+        sessions: [],
+    };
+
+    componentDidMount() {
+        API.get("api/askforhelp/all").then((res) => {
+            const afh = res.data;
+            this.setState({
+                sessions: afh,
+            });
+        });
+    }
+
+    openSubject = (subjectName) => {
+        this.setState({
+            currentTab: subjectName,
+        });
+    };
+
+    selectCheckbox = (title, onSelectSession) => {
+        return (
+            <input
```

Comment:
wanted to see if anyone had any ideas regarding the checkbox for each individual session.

Reply:
doing it this way is probably the best in terms of keeping things organized, but I think it has to be its own component rather than a function